diff --git a/node_modules/mxgraph/javascript/dist/build.js b/node_modules/mxgraph/javascript/dist/build.js
index 2ff1a45..5939383 100644
--- a/node_modules/mxgraph/javascript/dist/build.js
+++ b/node_modules/mxgraph/javascript/dist/build.js
@@ -8,7 +8,10 @@ root.mxgraph = factory();
 }
 }(this, function () {
 return function (opts) {
-for (var name in opts) { this[name] = opts[name]; }
+	console.log(this)
+	console.log("chage")
+	const self = window
+for (var name in opts) { self[name] = opts[name]; }
 var __mxOutput = {};
 /**
  * Copyright (c) 2006-2017, JGraph Ltd
